### ISSUE TITLE
Bump of packaging-21.3 to packaging-22.0 breaks cyclonedx-python

### ISSUE THREAD

**Issue by ssproessig-thales** [CycloneDX/cyclonedx-python#449]
```
Traceback (most recent call last):
  File "sources/env/bin/cyclonedx-py", line 5, in <module>
    from cyclonedx_py.client import main
  File "sources/env/lib/python3.10/site-packages/cyclonedx_py/client.py", line 37, in <module>
    from .parser.requirements import RequirementsParser
  File "sources/env/lib/python3.10/site-packages/cyclonedx_py/parser/requirements.py", line 31, in <module>
    from pip_requirements_parser import RequirementsFile  # type: ignore
  File "sources/env/lib/python3.10/site-packages/pip_requirements_parser.py", line 73, in <module>
    from packaging.version import LegacyVersion
ImportError: cannot import name 'LegacyVersion' from 'packaging.version' (sources/env/lib/python3.10/site-packages/packaging/version.py)
```

**Comment by radandreicristian** [comment]
Can confirm. For the moment, just pin packaging to v21.3.

**Comment by TheErk** [comment]
ditto here

**Comment by nudgegoonies** [comment]
Sounds like this needs code changes here:
https://github.com/pypa/packaging/issues/631

**Comment by a1lu** [comment]
There is a new version of pip_requirements_parser that fixes this upstream
https://github.com/nexB/pip-requirements-parser/pull/12/files

**Comment by jkowalleck** [comment]
re: https://github.com/CycloneDX/cyclonedx-python/issues/449#issuecomment-1345420178

why upgrade `pip-requirements-parser==32.0.0`? This project clearly defines: 
```
install_requires = \
['cyclonedx-python-lib>=2.0.0,<4.0.0',
 'packageurl-python>=0.9',
 'pip-requirements-parser>=31.2.0,<32.0.0',
 'setuptools>=47.0.0',
 'toml>=0.10.0,<0.11.0']
```

**Comment by jkowalleck** [comment]
thank you all for the input and your work.

a fix was released via [v3.7.3](https://github.com/CycloneDX/cyclonedx-python/releases/tag/v3.7.3)
